src/components/Quotes.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable */
+import { useState, useEffect } from "react";
```

Comment:
- [ ] It's highly advisable not to disable the linter in your coding project. Linters play a crucial role in maintaining code quality by enforcing coding standards and best practices. They help catch errors and potential bugs early in the development process, saving time and effort during debugging. Additionally, linters promote consistency in coding style across the project, making it easier for collaborators to work together efficiently. Keeping the linter enabled is a simple yet effective way to ensure your code remains clean, error-free, and easy to maintain.